Makefile

```
@@ -1,22 +1,44 @@
-.PHONY: build test test-integration generate install-paramgen
+.PHONY: build test test-integration generate install-paramgen install-mocken
 
-VERSION=$(shell git describe --tags --dirty --always)
+VERSION				=  $(shell git describe --tags --dirty --always)
+MOCKGEN_VERSION		?= v0.2.0
+PARAMGEN_VERSION	?= v0.7.2
+GOLANG_CI_LINT_VER	:= v1.54.2
 
+.PHONY: build
 build:
 	go build -ldflags "-X 'github.com/conduitio-labs/conduit-connector-weaviate.version=${VERSION}'" -o conduit-connector-weaviate cmd/connector/main.go
 
-test:
+.PHONY: install-mocken
```

probably a typo, it appears multiple times in the makefile
```suggestion
.PHONY: install-mockgen
```